pytest_girder/pytest_girder/plugin.py

```diff
@@ -8,5 +8,7 @@ def pytest_addoption(parser):
     group.addoption('--mongo-uri', action='store', default='mongodb://localhost:27017',
                     help=('The base URI to the MongoDB instance to use for database connections, '
                           'default is mongodb://localhost:27017'))
-    group.addoption('--drop-db', action='store', default='pre', choices=('pre', 'post', 'never'),
-                    help='When to destroy testing databases, default is pre (before running tests)')
+    group.addoption('--drop-db', action='store', default='both',
```

Comment:
I like this as the default for local dev testing, but could we make CI use `never`? It has the advantage of being faster, and preserving the databases for debugging in CI if we need to go poke around with SSH enabled.

Reply:
Would `pre` be better for CI?  If you are sshing in and rerunning the tests, then it would at least clear the old data?

Reply:
I think if you're SSHing in and re-running the tests, you'd be doing so manually, i.e. using the normal default of `both`.